test/unified_format.py

```diff
@@ -544,6 +544,19 @@ def setUpClass(cls):
             raise unittest.SkipTest(
                 '%s runOnRequirements not satisfied' % (cls.__name__,))
 
+        # add any special-casing for skipping tests here
+        if client_context.storage_engine == 'mmapv1':
+            if cls.TEST_SPEC['description'].find('retryable-writes') != -1:
```

Comment:
Nit: `'retryable-writes' in cls.TEST_SPEC['description']` instead of `find() != -1`

Reply:
Done